commands/sandbox.go

```diff
@@ -292,28 +302,26 @@ func deletePackage(c *CmdConfig, pkg string) error {
 
 // SandboxExec executes a sandbox command
 func SandboxExec(c *CmdConfig, command string, args ...string) (do.SandboxOutput, error) {
-	err := c.checkSandboxStatus()
+	err := c.checkSandboxStatus(c)
 	if err != nil {
 		return do.SandboxOutput{}, err
 	}
+	return sandboxExecNoCheck(c, command, args)
+}
+
+func sandboxExecNoCheck(c *CmdConfig, command string, args []string) (do.SandboxOutput, error) {
 	sandbox := c.Sandbox()
 	cmd, err := sandbox.Cmd(command, args)
 	if err != nil {
 		return do.SandboxOutput{}, err
 	}
-	// If DEBUG is specified, we need to open up stderr for that stream.  The stdout stream
-	// will continue to work for returning structured results.
-	if os.Getenv("DEBUG") != "" {
-		cmd.Stderr = os.Stderr
-	}
-
```

Comment:
This code was relocated to `sandbox.Cmd` where it will catch all paths that end up invoking the plugin.